src/XMakeBuildEngine/Resources/Strings.resx

```diff
@@ -1426,6 +1426,9 @@ Utilization:          {0} Average Utilization: {1:###.0}</value>
   <data name="OM_CannotRemoveMetadataOriginatingFromItemDefinition">
     <value>Cannot remove the metadata named "{0}" as it originates from an item definition, rather than being directly defined on this item.</value>
   </data>
+  <data name="OM_CannotSplitItemElementWhenSplittingIsDisabled">
+    <value>A project item operation tried to split the item element at location {0} but item element splitting is disabled with {1}.</value>
```

Comment:
The name `ThrowOnSplitItemElement` doesn't read to me like it's "disabling". Maybe this could say

> The requested operation cannot be completed without splitting the item element at location {0} into individual components, but item element splitting is disabled with {1}.

?